x/ibc/20-transfer/keeper/keeper_test.go

```diff
@@ -252,7 +253,7 @@ func (chain *TestChain) updateClient(client *TestChain) {
 
 func (chain *TestChain) createConnection(
 	connID, counterpartyConnID, clientID, counterpartyClientID string,
-	state ibctypes.State,
+	state connectiontypes.State,
```

Comment:
this is a nit but I noticed some imports are aliased as `connectiontypes` and some just `connection` and same for connection. Is there are reason for this? I think sticking to one or the other would be best

Reply:
yeah, maybe we should consider removing type aliases. We can do it in another PR 👍